velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -902,18 +896,31 @@ core::TypedExprPtr ExpressionFuzzer::generateArgFunction(const TypePtr& arg) {
     }
   }
 
+  core::TypedExprPtr body;
   if (eligible.empty()) {
-    return std::make_shared<core::LambdaTypedExpr>(
-        ROW(std::move(names), std::move(args)),
-        generateArgConstant(returnType));
+    body = generateArgConstant(returnType);
+    if (options_.referenceQueryRunner == nullptr ||
+        options_.referenceQueryRunner->isSupported(body)) {
+      return std::make_shared<core::LambdaTypedExpr>(
+          ROW(std::move(names), std::move(args)), body);
+    } else {
+      return std::make_shared<core::LambdaTypedExpr>(
+          ROW(std::move(names), std::move(args)),
+          generateArgColumn(returnType));
+    }
   }
 
   const auto idx = rand32(0, eligible.size() - 1);
   const auto name = eligible[idx];
 
+  if (name == "cast") {
```

Comment:
No, "cast" has always been available. The problem is that originally we generate a CallTypedExpr with the name being "cast". But this doesn't work with PrestoQueryRunner when converting this CallTypedExpr to SQL because "cast" requires a different syntax from regular functions. So this PR makes it to generate a proper CastTypedExpr for "cast".